workspace/assets/tasks/complexity.js

```diff
@@ -6,9 +6,6 @@ module.exports = function jshint (grunt) {
 		complexity: {
 			generic: {
 				src: '<%= src.js.src %>',
-				exclude: [
-					'./js/utils/keys.js'
-				],
```

Comment:
We need to keep this, because it will make project build fail....

Reply:
@fhamon ping ?

Reply:
Done and rebased.